test/test_create_db.py

```diff
@@ -0,0 +1,59 @@
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import unittest
+import os
+import sqlite3
+import pandas as pd
+from accounting import create_db as c
+
+
+class TestDB(unittest.TestCase):
+    # create database and make sure it exists
+    def test_00_test_create_db(self):
+        c.create_db("test/FluxAccounting.db")
```

Comment:
You will at least test the existence of the path. Better yet, perhaps `create_db` can do some more sanity check about the given path.